core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -67,6 +71,18 @@
     private static final Set<AsciiString> ADDITIONAL_HEADER_BLACKLIST = ImmutableSet.of(
             HttpHeaderNames.SCHEME, HttpHeaderNames.STATUS, HttpHeaderNames.METHOD, HttpHeaderNames.PATH);
 
+    private static final String ARTIFACT_ID = "armeria";
+    private static final String SERVER_NAME = "Armeria";
+    private static final String serverHeader;
+
+    static {
+        serverHeader = String.format("%s/%s (%s)",
+                                     SERVER_NAME,
+                                     Version.identify(HttpResponseSubscriber.class.getClassLoader())
+                                            .get(ARTIFACT_ID).artifactVersion(),
+                                     System.getProperty("os.name"));
```

Comment:
Had chat about `os.name` and we decided to remove `os.name` from `Server` header. Could you make `Server` header with `Armeria/{artifact_id}`? :-)

Reply:
Do you mean
```
Server: Armeria/armeria
```

or

```
Server: Armeria/0.94.1
```
It seems `Server: Armeria/{ARTIFACT_VERSION}` is corrent?

Reply:
yes the version. 😅

Reply:
Oops, artifact version!

Reply:
Thank you, fixed it :grinning: